app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -0,0 +1,42 @@
+package com.example.lhi06.residue;
+
+/**
+ * Created by lhi06 on 20/07/2017.
+ */
+
+public class AlbumReview {
+
+    Album thisAlbum;
```

Comment:
field should be name `album`. No reason to include `this` in the name.

Also rename the constructor parameter, setter method parameter and the getter/setter